### ISSUE TITLE
TypeError: Cannot read property 'elements' of undefined

### ISSUE THREAD

**Issue by livingtech** [GoogleForCreators/web-stories-wp#4181]
Basically we would love to use google web stories. but the plugin does not work with livingtech.ch

It drops this error regargless of what i do (e.g. insert image, click the text button, etc.)

TypeError: Cannot read property 'elements' of undefined
    in StoryProvider
    in Unknown
    in SnackbarProvider
    in Unknown
    in HistoryProvider
    in Unknown
    in Media3pApiProvider
    in Unknown
    in APIProvider
    in Unknown
    in ConfigProvider
    in LL
    in ThemeProvider
    in StyleSheetManager
    in App
    in FlagsProvider

**Comment by LuckynaSan** [comment]
@livingtech Thank you for reporting this issue. What browser and which version are you currently using, and on which operating system (Windows/Mac)?

Did this error prompt after you dropped the images or can you detail the steps you took before this issue occurred?

**Comment by livingtech** [comment]
this error promts after after nearly every action i take in the plugin. e.g. when i want to add a picture or when i want to add a headline.

i use windows 10 pro with chrome browser. i can check the browser version tomorrow but i guess its recent.

also the the dashboard is a plain white site.

we have wpmu installed but i tried to deactivate all those plugins but google web stories still did not work.

i could give u access if you access to the installation if you like!

cheers
fabian

Holen Sie sich Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: LuckynaSan <notifications@github.com>
Gesendet: Monday, August 24, 2020 6:50:14 PM
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <fabian.unruh@livingtech.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


@livingtech<https://github.com/livingtech> Thank you for reporting this issue. What browser and which version are you currently using, and on which operating system (Windows/Mac)?

Did this error prompt after you dropped the images or can you detail the steps you took before this issue occurred?

**Comment by LuckynaSan** [comment]
Thank you for sharing those details. We cannot have access, but it will be great if you could provide the browser version when you can.

- Did you create a new story or use an existing template? 
- Can you share your [Site Health](https://i.imgur.com/Rtkmsba.png) information? 
- A screenshot or video can be helpful as well. You can use software like [Jumpshare](https://jumpshare.com/) to make a screenshot or https://recordit.co/ to make a video and share the link.

**Comment by livingtech** [comment]
I recorded my screen: https://drive.google.com/file/d/1nQcB6iirc8Iygj8qFXArbUcU7HV3bWYP/view

Hope this helps. Below is my browser version.

Cheers
Fabian

**Comment by LuckynaSan** [comment]
@livingtech Thank you for sharing your video. 

What version of Chrome are you using?

Could you please share your Site Health info by navigating to Info then [Copy site info to clipboard](https://i.imgur.com/Rtkmsba.png) and pasting it here?

**Comment by livingtech** [comment]
Here you go! Hope this helps.


[cid:image001.png@01D67D55.D80644F0]



[cid:image002.png@01D67D55.D80644F0]

**Comment by LuckynaSan** [comment]
It appears the images you attempted to share are not displaying here. Are you using Chrome version 84? Can you paste your Site Health here after copying it to the clipboard?

**Comment by livingtech** [comment]
Oh, ok… i just copied them into the mail…

Chrome 85.0.4183.83

Wordpress 5.4.2



Site-Health-Info:
1 critical issue
Background updates are not working as expectedSecurity
Background updates ensure that WordPress can auto-update if a security update is released for the version you are currently using.

  *   Error The WP_AUTO_UPDATE_CORE constant is defined and enabled.
  *   Passed No version control systems were detected.
  *   Passed Your installation of WordPress doesn't require FTP credentials to perform updates.
  *   Passed All of your WordPress files are writable.
  *   Error WordPress security and maintenance releases are blocked by define( 'WP_AUTO_UPDATE_CORE', false );.
3 recommended improvements
WordPress update available (5.5)Performance
A new version of WordPress is available.
Install the latest version of WordPress<https://www.livingtech.ch/wp-admin/update-core.php>
You should remove inactive pluginsSecurity
Plugins extend your site’s functionality with things like contact forms, ecommerce and much more. That means they have deep access to your site, so it’s vital to keep them up to date.
Your site has 2 plugins waiting to be updated.
Your site has 6 inactive plugins. Inactive plugins are tempting targets for attackers. If you’re not going to use a plugin, we recommend you remove it.
Manage your plugins<https://www.livingtech.ch/wp-admin/plugins.php>
Update your plugins<https://www.livingtech.ch/wp-admin/plugins.php?plugin_status=upgrade>
Manage inactive plugins<https://www.livingtech.ch/wp-admin/plugins.php?plugin_status=inactive>
Have a default theme availableSecurity
Themes add your site’s look and feel. It’s important to keep them up to date, to stay consistent with your brand and keep your site secure.
Your site has 1 installed theme, and it is up to date.
Your site does not have any default theme. Default themes are used by WordPress automatically if anything is wrong with your normal theme.
Manage your themes<https://www.livingtech.ch/wp-admin/themes.php>



Von: LuckynaSan <notifications@github.com>
Gesendet: Freitag, 28. August 2020 16:28
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <Removed email>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


It appears the images you attempted to share are not displaying here. Are you using Chrome version 84? Can you paste your Site Health here after copying it to the clipboard?

**Comment by livingtech** [comment]
And here is the rest of Site Health Info:

```
### wp-core ###

version: 5.4.2
site_language: de_CH
user_language: en_US
timezone: +01:00
permalink: /%postname%/
https_status: true
user_registration: 0
default_comment_status: open
multisite: false
user_count: 3
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /srv/www/vhosts/livingtech.ch/httpdocs
wordpress_size: 65.04 MB (68197896 bytes)
uploads_path: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/uploads
uploads_size: 779.74 MB (817612509 bytes)
themes_path: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/themes
themes_size: 39.68 MB (41612438 bytes)
plugins_path: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/plugins
plugins_size: 172.77 MB (181157248 bytes)
database_size: 27.14 MB (28461148 bytes)
total_size: 1.06 GB (1137041239 bytes)

### wp-dropins (1) ###

advanced-cache.php: true

### wp-active-theme ###

name: LivingTech (refive-child)
version: 1.0
author: LivingTech GmbH
author_website: http://www.livingtech.ch
parent_theme: LivingTech (refive)
theme_features: widgets, qode_sidebar, menus, post-formats, automatic-feed-links, post-thumbnails
theme_path: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/themes/refive-child

### wp-parent-theme ###

name: LivingTech (refive)
version: 1.0
author: LivingTech GmbH
author_website: http://www.livingtech.ch
theme_path: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/themes/refive

### wp-plugins-active (20) ###

Advanced Custom Fields: version: 5.9.0, author: Elliot Condon
Beehive Pro: version: 3.2.8, author: WPMU DEV
BIALTY - Bulk Image Alt Text (Alt tag, Alt Attribute) with Yoast SEO + WooCommerce: version: 1.4.3, author: Pagup
Defender Pro: version: 2.3.1, author: WPMU DEV
GDPR Cookie Consent Banner: version: 2.3.15, author: termly
Head, Footer and Post Injections: version: 3.2.1, author: Stefano Lissa
Header and Footer Scripts: version: 2.2.1, author: Digital Liberation
Loco Translate: version: 2.4.2, author: Tim Whitlock (latest version: 2.4.3)
Quform: version: 2.9.3, author: ThemeCatcher (latest version: 2.12.0)
Quick Page/Post Redirect Plugin: version: 5.2.0, author: anadnet
Slider Revolution: version: 6.2.17, author: ThemePunch
SmartCrawl Pro: version: 2.7.0, author: WPMU DEV
SMTP2GO: version: 1.0.6, author: SMTP2GO
Visualizer: Tables and Charts for WordPress: version: 3.4.6, author: Themeisle
Web Stories: version: 1.0.0-beta.2, author: Google
WP-SCSS: version: 1.2.5, author: Connect Think
WPBakery Page Builder: version: 6.3.0, author: Michael M - WPBakery.com
WPMU DEV Dashboard: version: 4.10.1, author: WPMU DEV
WP Rocket: version: 3.6.4, author: WP Media
WPS Hide Login: version: 1.5.6, author: WPServeur, NicolasKulka, tabrisrp

### wp-plugins-inactive (6) ###

ACF Content Analysis for Yoast SEO: version: 2.6, author: Thomas Kräftner, ViktorFroberg, marol87, pekz0r, angrycreative, Team Yoast
ChatBot: version: 3.4.8, author: QuantumCloud
LT Backup: version: 1.0, author: Theodor Nedelea - LivingTech GmbH
Simple Custom CSS and JS: version: 3.33, author: SilkyPress.com
Snapshot Pro: version: 3.3.0, author: WPMU DEV
Tidio Chat: version: 4.2.1, author: Tidio Ltd.

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1687
imagemagick_version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 http://www.imagemagick.org
imagick_limits:
                imagick::RESOURCETYPE_AREA: 71 GB
                imagick::RESOURCETYPE_DISK: 1.844674407371E+19
                imagick::RESOURCETYPE_FILE: 6144
                imagick::RESOURCETYPE_MAP: 71 GB
                imagick::RESOURCETYPE_MEMORY: 35 GB
                imagick::RESOURCETYPE_THREAD: 6
gd_version: bundled (2.1.0 compatible)
ghostscript_version: unknown

### wp-server ###

server_architecture: Linux 4.9.0-12-amd64 x86_64
httpd_software: Apache
php_version: 7.4.8 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 240
memory_limit: 512M
max_input_time: 60
upload_max_size: 256M
php_post_max_size: 64M
curl_version: 7.52.1 OpenSSL/1.0.2u
suhosin: false
imagick_availability: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.1.44-MariaDB-0+deb9u1
client_version: mysqlnd 7.4.8

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content
WP_PLUGIN_DIR: /srv/www/vhosts/livingtech.ch/httpdocs/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8mb4
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

```

**Comment by LuckynaSan** [comment]
I appreciate your quick response! Could you attempt deactivating [SmartCrawler](https://wordpress.org/plugins/smartcrawl-seo/) by WPMU DEV plugin then open the Web Stories editor once more? Let us know if the issue persists.

**Comment by livingtech** [comment]
Hi!

Yes this seemed to help. I can now create stories without any problems.

The dashboard is still plain white with no content though. I see the content loading but that a white overlay appears. Maybe some z-index problem?

Cheers
Fabian

**Comment by swissspidy** [comment]
> The dashboard is still plain white with no content though. I see the content loading but that a white overlay appears. Maybe some z-index problem?

That sounds more like a JavaScript error. Do you see any errors in the browser console? A screenshot would help.

Please upload screenshots via the GitHub website. Email attachments don't upload correctly.

**Comment by livingtech** [comment]
You are propabely right 😊


[cid:image001.png@01D67F82.64B751E0]


[cid:image002.png@01D67F82.64B751E0]




Von: Pascal Birchler <notifications@github.com>
Gesendet: Montag, 31. August 2020 10:33
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <fabian.unruh@livingtech.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


The dashboard is still plain white with no content though. I see the content loading but that a white overlay appears. Maybe some z-index problem?

That sounds more like a JavaScript error. Do you see any errors in the browser console? A screenshot would help.

Please upload screenshots via the GitHub website. Email attachments don't upload correctly.

**Comment by swissspidy** [comment]
@livingtech I am afraid email attachments don't upload correctly. Please upload screenshots via the GitHub website.

**Comment by livingtech** [comment]
Sorry…

Heres the console (dev tools)

stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326 TypeError: Cannot read property 'id' of null
    at stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:318
    at Array.map (<anonymous>)
    at LB (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:318)
    at Do (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at Ti (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at Qj (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at Kj (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at _a (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326
    at t.unstable_runWithPriority (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)

Ci @ stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326
stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326 TypeError: Cannot read property 'title' of null
    at stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:318
    at Array.map (<anonymous>)
    at stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:318
    at Object.Mh [as useMemo] (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at t.useMemo (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at TypeaheadSearch (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:318)
    at Do (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at ai (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at Ti (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)
    at Qj (stories-dashboard.js?ver=6c93f87dbd86f0935e6c1842a68e76b5:326)






Von: Pascal Birchler <notifications@github.com>
Gesendet: Montag, 31. August 2020 10:41
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <fabian.unruh@livingtech.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


@livingtech<https://github.com/livingtech> I am afraid email attachments don't upload correctly. Please upload screenshots via the GitHub website.

**Comment by LuckynaSan** [comment]
@livingtech Could try deactivating [WPBakery Page Builder](https://wpbakery.com/) and attempt to access the Web Stories dashboard?

**Comment by livingtech** [comment]
Hi!

No, deactivating wpbakery page builde has no influence.

But it worked to deactivate smart crawler pro. Unfortunately we need smart crawler as it generates sitemap.xml. I can deactivate it, create a story and activate it again though 😊

Cheers


Von: LuckynaSan <notifications@github.com>
Gesendet: Dienstag, 1. September 2020 19:32
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <fabian.unruh@livingtech.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


@livingtech<https://github.com/livingtech> Could try deactivating WPBakery Page Builder<https://wpbakery.com/> and attempt to access the Web Stories dashboard?

**Comment by ernee** [comment]
@livingtech we'll be tracking compatibility with SmartCrawl SEO [in this issue](https://github.com/google/web-stories-wp/issues/4491). With the most recent pro version 2.8.0, the error initially reported does not occur in my testing.

Are you able to update that plugin and let us know if you still experience any issues? Thanks!

**Comment by livingtech** [comment]
Yes, you seem to be right. I have 2.8.0, and the plugin works fine.
Only thing that is not working is that I still get a plain white dashboard.

Cheers
Fabian

Von: Renee Johnson <notifications@github.com>
Gesendet: Dienstag, 15. September 2020 19:02
An: google/web-stories-wp <web-stories-wp@noreply.github.com>
Cc: Fabian Unruh <fabian.unruh@livingtech.ch>; Mention <mention@noreply.github.com>
Betreff: Re: [google/web-stories-wp] TypeError: Cannot read property 'elements' of undefined (#4181)


@livingtech<https://github.com/livingtech> we'll be tracking compatibility with SmartCrawl SEO in this issue<https://github.com/google/web-stories-wp/issues/4491>. In the most recent pro version 2.8.0 and the error initially reported does not occur in my testing.

Are you able to update that plugin and let us know if you still experience any issues? Thanks!

**Comment by LuckynaSan** [comment]
The stable version of the plugin 1.0.0 as been released. If you haven't already, please [download](https://wp.stories.google/) that version. We will be closing this topic out and provide support for the plugin in the [support forum](https://wordpress.org/support/plugin/web-stories/). Please feel free to post a topic about any additional issues you encounter there. Thanks!
